terraform/lambda.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_lambda_function" "mattermost_figma_lambda_function" {
+  runtime          = "${var.lambda_runtime}"
+  function_name = "mattermost_figma_lambda_function"
+  s3_bucket = "${var.figma_aws_lambda_bucket}"
```

Comment:
@prokhorind I imagine we'll want to avoid project-specific names in the configurations as much as possible. This way we can avoid doing "find and replace" when we apply the same configs to other projects.